### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by AlexJuca** [josemarjobs/sensor_app#1]
Is this code based on this course? -> https://www.pluralsight.com/courses/go-build-distributed-applications

**Comment by josemarjobs** [comment]
Yes, it is. Really good course, fast passed but full of awesome info.

On Dec 30, 2016 7:30 AM, Alexandre Antonio Juca <notifications@github.com> wrote:

Is this code based on this course? -> https://www.pluralsight.com/courses/go-build-distributed-applications

**Comment by AlexJuca** [comment]
Can you share the course with me please :)?

2016-12-30 17:31 GMT+01:00 Josemar da Costa Magalhaes <
notifications@github.com>:
